backend/inspirehep/migrator/cli.py

```diff
@@ -156,3 +157,41 @@ def continuously():
             touch_file(liveness_file)
         continuous_migration()
         sleep(current_app.config.get("MIGRATION_POLLING_SLEEP", 1))
+
+
+@migrate.command()
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    default=False,
+    help="Force the task to run even in debug mode.",
+)
+@click.option(
+    "-w",
+    "--wait",
+    is_flag=True,
+    default=False,
+    help="Wait for all subtasks to finish. (No progress bar will be available still)",
+)
+@click.option(
+    "-s",
+    "--step",
+    default=0,
+    type=int,
+    help="""
+    1 - Migrate from mirror table
```

Comment:
this will not have a new line on help I think

Reply:
This is how  it looks like:
```
[pazembrz@pawel-pc backend]$ poetry run inspirehep migrate mirror_step --help
Usage: inspirehep migrate mirror_step [OPTIONS]

  Migrate records step by step

  (Re)Migrates only records marked as valid

Options:
  -f, --force         Force the task to run even in debug mode.
  -w, --wait          Wait for all subtasks to finish. (No progress bar will
                      be available still)
  -s, --step INTEGER  1 - Migrate from mirror table
                      2 - Recalculate citations
                      proceedings etc.
                      3 - Reindex
  --help              Show this message and exit.
```